src/Mizar/SamplingWithFrameTrackInputWidget.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_SAMPLING_WITH_FRAME_TRACK_INPUT_WIDGET_H_
+#define MIZAR_SAMPLING_WITH_FRAME_TRACK_INPUT_WIDGET_H_
+
+#include <QWidget>
+#include <memory>
+#include <string_view>
+
+#include "MizarData/MizarPairedData.h"
+
+namespace Ui {
+class SamplingWithFrameTrackInputWidget;
```

Comment:
I am probably missing some Qt specifics for custom Widgets, but this forward declaration here looks strange.

Reply:
It's a requirement for the pimpl idiom.